### PR TITLE
PYIC-3932: Added reset-identity event to reuse page

### PR DESCRIPTION
**Pull request by RogVanGDS**
========== DO NOT MERGE IN UNLESS READY TO GO LIVE ===========

### What changed

Added reset-identity event to reuse page

### Why did it change

To allow core-front to route from reuse-identity page into the user initiated VC reset flow

### Issue tracking

- [PYIC-3932](https://govukverify.atlassian.net/browse/PYIC-3932)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

[PYIC-3932]: https://govukverify.atlassian.net/browse/PYIC-3932?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Joe-Edwards-GDS — APPROVED** [review]
Looks sensible, but might need to wait until next week to merge, so we don't interrupt the load test.

**Comment by RogVanGDS** [comment]
> Looks sensible, but might need to wait until next week to merge, so we don't interrupt the load test.

Yep, this is essentially the go-live for this whole feature, so we won't be merging in until the email trigger/other dependent tickets are ready anyway

**Comment by Joe-Edwards-GDS** [comment]
Closing as this is now superseded by the identity continuity initiative

**Pull request closed without merging**
